contrib/generate-release.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# pylint: disable=invalid-name,missing-docstring
+
+import subprocess
+import datetime
+
+
+def _get_last_release() -> str:
+    p = subprocess.run(
+        ["git", "describe", "--tags", "--abbrev=0"], check=True, capture_output=True
+    )
+    return p.stdout.decode().replace("\n", "")
+
+
+def _get_next_release(last_tag: str) -> str:
+    triplet: list[str] = last_tag.split(".")
+    return f"{triplet[0]}.{triplet[1]}.{int(triplet[2])+1}"
+
+
+def _get_appstream_date() -> str:
+    return datetime.datetime.now().strftime("%Y-%m-%d")
+
+
+def _generate_release_notes(last_tag: str, next_tag: str) -> str:
+    args = [
+        "git",
+        "log",
+        "--format=%s",
+        "--cherry-pick",
```

Comment:
Why need this one?

Reply:
For this case I don't -- not `--right-only` -- both removed.